src/binary-reader-objdump.c

```diff
@@ -35,6 +35,11 @@ typedef struct Context {
   WasmBool print_details;
   WasmBool header_printed;
   int section_found;
+
+  WasmStringSlice import_module_name;
+  WasmStringSlice import_field_name;
+
+  int function_index;
```

Comment:
OK for now, but this will be needed for other imported types too (globals, tables, memories).